pycaret/clustering/oop.py

```diff
@@ -46,8 +45,10 @@ def _get_models(self, raise_errors: bool = True) -> Tuple[dict, dict]:
             ).items()
             if not v.is_special
         }
-        all_models_internal = pycaret.containers.models.clustering.get_all_model_containers(
-            self, raise_errors=raise_errors
+        all_models_internal = (
```

Comment:
same comment as before - is `black` autoformatting this for you? Seems odd though.

Reply:
Not exactly sure. It didn't _look_ like an issue, although I did find it odd at first as well that it did it automatically. The changes appeared as separate changes after I staged my own.

Reply:
ok, no problem. Looks like the autoformatter at work. The tests are passing so it should be ok.